examples/set-theory-domain/venn-color.sty

```diff
@@ -0,0 +1,99 @@
+-- TODO: Just optimize locations in 1D for now, and `r` proportional to `x` location
```

Comment:
I think I am going to cherry-pick the color optimization commit onto another branch and revert it on this branch. We'll see what it does to the commit history

Reply:
I just reverted it here and made another branch (https://github.com/penrose/penrose/tree/color-opt) w/o the revert commit. Not sure what will happen when `color-opt` gets merged--do we revert the revert commit? LMK if you know of a better way.

Reply:
Not sure if I'm understanding correctly. I wouldn't mind keeping the changes, as long as you have it on another branch. Keeping the changes might actually be nicer for the merge in the future?